common/search/src/main/scala/weco/api/search/config/builders/PipelineElasticClientBuilder.scala

```diff
@@ -6,15 +6,57 @@ import software.amazon.awssdk.services.secretsmanager.model.GetSecretValueReques
 import weco.catalogue.display_model.PipelineClusterElasticConfig
 import weco.elasticsearch.ElasticClientBuilder
 
+import scala.util.Try
+
+/**
+  * Build an ElasticClient that will connect to the appropriate database
+  * as specified for this environment.
+  *
+  * The settings may come either from environment variables
+  * (if API_ELASTIC_SETTINGS_FROM_ENVIRONMENT is true)
+  * or from the AWS Secrets Manager
+  */
 object PipelineElasticClientBuilder {
+  def apply(serviceName: String): ElasticClient =
+    sys.env.get("API_ELASTIC_SETTINGS_FROM_ENVIRONMENT") match {
```

Comment:
As discussed, let's change this to something like `API_SETTINGS_MODE` which takes value `environment|secretsmanager`, and fails to start if it gets an unrecognised value.